services/QuillLMS/spec/spec_helper.rb

```diff
@@ -17,6 +19,14 @@
   end
 end
 
+Geocoder::Lookup::Test.set_default_stub(
+  [
+    {
+      timezone: 'America/New_York'
+    }
+  ]
+)
+
```

Comment:
Nice job configuring this.

Reply:
Agreed.  Sometimes the hardest part is getting the spec configured properly. 

Some notes on conventions:

`spec_helper.rb` is normally reserved for configuring specs that are not rails dependent (more info [here](https://stackoverflow.com/a/24145760))

For rspec configuration other than general configuration that applies to every spec, a convention is to place gem configuration in a new file within spec/support (e.g. [carrierwave.rb](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/spec/support/carrierwave.rb)

So for this just create a new file `spec/support/geocoder.rb`: 
``` 
Geocoder.configure(lookup: :test, ip_lookup: :test)
Geocoder::Lookup::Test.set_default_stub([{ timezone: 'America/New_York' }])
```

This will automatically get pulled via rails_helper.rb using this [line](https://github.com/empirical-org/Empirical-Core/blob/cbcc5133d404bca494601add99a7d546aeab2271/services/QuillLMS/spec/rails_helper.rb#L43)